File: org.yamcs.studio.examples/examples/BOY Examples/Miscellaneous/LoginDialogExample/OPILogin.py
from java.lang import System

ok = PVUtil.getDouble(pvs[0])
if ok == 1:
    userName = System.getProperty("UserName")
    password = System.getProperty("Password")
    if userName=="admin" and password == "123456":
        widget.setPropertyValue("visible", True)
    else:
        GUIUtil.showErrorDialog("The user name or password is wrong!")
        pvs[0].setValue(0)

File: org.yamcs.studio.examples/examples/OPI Examples/Miscellaneous/LoginDialogExample/OPILogin.js
var System = Java.type("java.lang.System");

var ok = PVUtil.getDouble(pvs[0]);
if (ok == 1) {
	var username = System.getProperty("UserName");
	var password = System.getProperty("Password");
	if (username == "admin" && password == "123456") {
		widget.setPropertyValue("visible", true);
	} else {
		GUIUtil.openErrorDialog("The user name or password is wrong!");
		pvs[0].setValue(0);
	}
}
